lib/internal/bootstrap/node.js

```diff
@@ -611,6 +611,24 @@
       // It's possible that defaultTriggerAsyncId was set for a constructor
       // call that threw and was never cleared. So clear it now.
       clearDefaultTriggerAsyncId();
+      // if node-report is enabled, call into its handler to see
+      // whether it is interested in handling the situation.
+      // Ignore if the error is scopped inside a domain.
+      if (er.domain === undefined || er.domain == null) {
```

Comment:
I think the idea was to have `if (er.domain == null)` as it will check for both `null` and `undefined` here and in other places.
Though I personally don't have a strong opinion on either way (`(val === undeifned || val === null)` or `(val == null)`).